src/engine/Sim3D.ts

```diff
@@ -167,6 +172,13 @@ export class Sim3D extends EventEmitter {
     this.isRendering = false;
   }
 
+  /**
+   * Halt physics from re-processing, leaving rendering to continue
+   */
+  haltPhysics(): void {
```

Comment:
Either add a corresponding `startPhysics` call (if you do this, rename this to `stopPhysics` maybe), or make this `setPhysicsActive(physActive: boolean )`